src/Tizen.Network.Stc/Tizen.Network.Stc/StcRule.cs

```diff
@@ -0,0 +1,288 @@
+            if(ret != (int)StcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to create Rule handle, Error - " + (StcError)ret);
+                StcErrorFactory.ThrowStcException(ret);
+            }
+        }
+
+        /// <summary>
+        /// Destroy the StcRule object
+        /// </summary>
+        ~StcRule()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// A method to destroy the managed StcRule objects.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        protected void Dispose(bool disposing)
+        {
+            if (_disposed)
+            {
+                return;
+            }
+
+            if(disposing)
+            {
+                // destroy managed resources
+            }
+
+            int ret = Interop.Stc.Rule.Destroy(_ruleHandle);
+            if (ret == (int)StcError.None)
+            {
+                _ruleHandle = IntPtr.Zero;
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Failed to destroy Rule handle, Error - " + (StcError)ret);
+            }
+
+            _disposed = true;
+        }
+
+        /// <summary>
+        /// A property for Application ID for statistics rule.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>AppId.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public string AppId
+        {
+            get
+            {
+                string appId;
+                int ret = Interop.Stc.Rule.GetAppId(_ruleHandle, out appId);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get AppId, Error - " + (StcError)ret);
+                    return string.Empty;
+                }
+                return appId;
+            }
+            set
+            {
+                if (_disposed)
+                {
+                    throw new ObjectDisposedException("Invalid StcRule instance (Object may have been disposed or released)");
+                }
+                int ret = Interop.Stc.Rule.SetAppId(_ruleHandle, value);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to set AppId, Error - " + (StcError)ret);
+                    StcErrorFactory.ThrowStcException(ret);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Set the time interval for statistics rule.
```

Comment:
`Interval` seem be little ambiguus, IMO, Range is more proper for it

Reply:
Done.